### ISSUE TITLE
breadcrumb links don't work with persistent URLs

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#163]
When trying to view pages using the ChronAm-style persistent URLs, the page throws an error:
```
NoMethodError in NewspaperWorks::Newspapers#page
Showing /home/vagrant/newspaper_works/app/views/newspaper_works/base/_newspaper_hierarchy.html.erb where line #11 raised:
undefined method `next_page_link' for #<#<Class:0x00000000088dce50>:0x00007f1f7c8f1ba8>
```
This is because the breadcrumb methods in `NewspaperWorks::BreadcrumbHelper` and `NewspaperPagesHelper` are not accessible by default to `NewspaperWorks::NewspapersController` which is responsible for handling the persistent URL requests.

This should be fairly easy to fix:
1. Create a `app/helpers/newspaper_works/newspapers_helper.rb` file:
```
module NewspaperWorks
  module NewspapersHelper
    include NewspaperWorks::BreadcrumbHelper
  end
end
```
2. Move the `#previous_page_link` and `#next_page_link` methods from `NewspaperPagesHelper` into `NewspaperWorks::BreadcrumbHelper`. (This is fine, since these methods are well within the scope of `BreadcrumbHelper`.)

3. Adjust specs accordingly.

**Comment by ebenenglish** [comment]
There is a related issue with the display of Title pages via the persistent URLs.

In `app/views/hyrax/newspaper_titles/show.html.erb`, we need to add the "full"-ish path to the `issue_search_form` and `all_front_pages_form` partials, so that they can be rendered from NewspapersController.

This:
```
<%= render partial: "issue_search_form", presenter: @presenter %>
<%= render partial: "all_front_pages_form", presenter: @presenter %>
```
Should be something like:
```
<%= render partial: "hyrax/newspaper_titles/issue_search_form", presenter: @presenter %>
<%= render partial: "hyrax/newspaper_titles/all_front_pages_form", presenter: @presenter %>
```

**Comment by ebenenglish** [comment]
Fixed via #173.
